File: packages/ios/Sources/CivUI/CivSideNav.swift
// CivUI — CivSideNav for SwiftUI
// Vertical hierarchical left-rail navigation. Mirrors civ-side-nav.
//
// Placeholder body — the prop surface satisfies schema parity. See
// .claude/rules/audit-debt.md → "Native platform implementation pass".

import SwiftUI

public struct CivSideNav: View {
    /// Accessible label for the rail (announced as a `<nav>` landmark).
    public var label: String

    /// Tap-target density. `default` keeps the compact rail; `lg`
    /// raises every row to the WCAG 2.5.5 AAA 44pt floor.
    public var spacing: String

    /// Typographic weight. `secondary` keeps the quiet sub-nav
    /// treatment; `primary` swaps in bold body text to match civ-nav.
    public var emphasis: String

    public init(label: String = "", spacing: String = "default", emphasis: String = "secondary") {
        self.label = label
        self.spacing = spacing
        self.emphasis = emphasis
    }

    public var body: some View {
        EmptyView()
    }
}

File: packages/android/src/main/kotlin/gov/civui/components/CivSideNav.kt
// CivUI — CivSideNav for Jetpack Compose
// Vertical hierarchical left-rail navigation. Mirrors civ-side-nav.
//
// Placeholder body — see .claude/rules/audit-debt.md → "Native platform implementation pass".

package gov.civui.components

import androidx.compose.foundation.layout.Column
import androidx.compose.runtime.Composable
import androidx.compose.ui.Modifier

@Composable
fun CivSideNav(
    label: String = "",
    spacing: String = "default",
    emphasis: String = "secondary",
    modifier: Modifier = Modifier,
    content: @Composable () -> Unit = {},
) {
    Column(modifier = modifier) {
        // Placeholder — see audit-debt.md for native implementation status.
        content()
    }
}
